amplify/data/resource.ts

```diff
@@ -80,18 +76,56 @@ const schema = a
         allow.group("Admin"),
         allow.authenticated().to(["read"]),
       ]),
-
     Team: a
       .model({
         name: a.string(),
         id: a.id(),
         approved: a.boolean(),
         members: a.hasMany("User", "teamId"),
+        scores: a.hasMany("Score", "teamId"),
+        teamRooms: a.hasMany("TeamRoom", "teamId"),
```

Comment:
Are there multiple team rooms because teams are getting judged more than once by different judges?

Reply:
Yeah that's a possibility that could happen